hazelcast-client/src/test/java/com/hazelcast/client/ClientClusterRestartEventTest.java

```diff
@@ -144,11 +147,23 @@ public void memberAttributeChanged(MemberAttributeEvent memberAttributeEvent) {
             }
         });
 
-        instance1.shutdown();
-        instance2.shutdown();
+        instance1.getCluster().shutdown();
 
-        instance1 = hazelcastFactory.newHazelcastInstance(newConfig());
-        instance2 = hazelcastFactory.newHazelcastInstance(newConfig());
+        Future<HazelcastInstance> f1 = spawn(new Runnable() {
```

Comment:
Should we make this Callable not to get NPE on line 168 ?

Reply:
Opps A mistake when backporting from java 8 codebase to maint. Fixed it.